libs/utils/utils.js

```diff
@@ -1126,6 +1126,12 @@ export async function loadDeferred(area, blocks, config) {
     import('../features/personalization/preview.js')
       .then(({ default: decoratePreviewMode }) => decoratePreviewMode());
   }
+  if (config?.dynamicNavKey && config?.env?.name !== 'prod') {
+    const { miloLibs } = getConfig();
```

Comment:
Redundant getConfig and unnecessary optional chaining.
```suggestion
  if (config.dynamicNavKey && config.env?.name !== 'prod') {
    const { miloLibs } = config;
```